admin/src/react-components/accounts.js

```diff
@@ -168,10 +169,10 @@ export const AccountList = withStyles(styles)(
         }, 6000);
       }
       render() {
-        const { classes } = this.props;
+        const { classes, refreshView, ...other } = this.props;
         return (
           <>
-            <Card classes={{ root: classes.searchCard }}>
+            <Card className={classes.searchCard}>
```

Comment:
Hmm why does this change from `classes` to `className`? Quick search shows that some react-admin stuff expects `classes`, not sure if this is one of them. Possible it was just wrong before. Did you notice it breaking something?

Reply:
Double checked, both ways work. I've worked with material UI before and this is the simpler preferred way - no errors in console, style is applied!